Core/Libraries/Source/WWVegas/WWLib/mempool.h

```diff
@@ -157,7 +157,7 @@ class AutoPoolClass
 ** Macro to declare the allocator for your class.  Put this in the cpp file for
 ** the class.
 */
-#if defined(_MSC_VER) && _MSC_VER < 1300
```

Comment:
Are we sure this is correct and that we are checking if the macro is none zero rather than if its defined?